### PR TITLE
Fix zeros stripped in conversion from numeric

### PR DESCRIPTION
**Pull request by michal-tomlein**
Fixes a bug causing zeros to be stripped from numeric values such as 10234.543201, producing 1234.54321.

### CODE REVIEW

File: Sources/PostgreSQL/Data/PostgreSQLData+String.swift [hunk 1]
```diff
@@ -48,10 +48,10 @@ extension String: PostgreSQLDataConvertible {
                     
                     /// depending on our offset, append the string to before or after the decimal point
                     if offset < metadata.weight.bigEndian + 1 {
-                        integer += string
+                        integer += offset == 0 ? string : String(repeating: "0", count: 4 - string.count) + string
```
Comment (tanner0101) [inline Sources/PostgreSQL/Data/PostgreSQLData+String.swift]: Breaking this up into one or two lines with some comments would make it a lot more readable I think

**Comment by tanner0101** [comment]
I would love to include this change in 1.1.1. Let me know if you have any questions about writing the test. I'd also be happy to help write one.

**Comment by michal-tomlein** [comment]
I can look into writing the test tomorrow (it’s late here). To be honest I don’t think a test should delay the merge of a critical data corruption bug fix. I could open a separate pull request.

**Comment by penny-coin** [comment]
Hey @michal-tomlein, you just merged a pull request, have a coin! 

You now have 1 coins.

**Pull request merged**